src/main/java/com/google/firebase/auth/FirebaseAuth.java

```diff
@@ -217,12 +221,47 @@ public FirebaseToken call() throws Exception {
 
         // This will throw a FirebaseAuthException with details on how the token is invalid.
         firebaseTokenVerifier.verifyTokenAndSignature(firebaseToken.getToken());
-
+        
+        if (checkRevoked) {
+          String uid = firebaseToken.getUid();
+          UserRecord user = userManager.getUserById(uid);
+          if (user.getTokensValidAfterTimestamp() 
+              > ((long)firebaseToken.getClaims().get("iat")) * 1000) {
+                throw new FirebaseAuthException("id-token-revoked", "Firebase auth token revoked");
+          }        
+        }       
         return firebaseToken;
       }
     });
   }
 
+  private Task<Void> revokeRefreshTokens(String uid) {
+    checkNotDestroyed();  
+    final UpdateRequest request = new UpdateRequest(uid).setValidSince(new Date().getTime() / 1000);
+    return call(new Callable<Void>() {
+      @Override
+      public Void call() throws Exception {
+        userManager.updateUser(request, jsonFactory);
+        return null;
+      }
+    });
+  }
+
+  /**
+   * Revokes all refresh tokens for the specified user.
+   * 
+   * <p>In addition to revoking all refresh tokens for a user, all ID tokens issued
+   * before revocation will also be revoked at the Auth backend. Any request with an
```

Comment:
I believe this is incorrect. Please align with the documentation of Node.js SDK.